src/app/routes/utils/pageTypes.js

```diff
@@ -12,3 +12,4 @@ export const PHOTO_GALLERY_PAGE = 'PGL';
 export const CORRESPONDENT_STORY_PAGE = 'CSP';
 export const TOPIC_PAGE = 'TOPIC';
 export const LIVE_PAGE = 'LIVE';
+export const HOME_PAGE = 'HOME';
```

Comment:
Similar to above, I feel like we'll be using the `FRONT_PAGE` value for this page type as it will be a direct swap with the existing page. Fine for now though!

Reply:
You're right, we know we will have to come back and change this later!